mmv1/products/netapp/activeDirectory.yaml

```diff
@@ -0,0 +1,187 @@
+    skip_test: true
+    vars:
+      active_directory_basic_name: 'test-active-directory-basic'
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'netapp_active_directory_full'
+    primary_resource_id: 'test_active_directory_full'
+    skip_test: true
+    vars:
+      active_directory_full_name: 'test-active-directory-full'
+properties:
+  # Fields go here
+  - !ruby/object:Api::Type::String
+    name: name
+    output: true
+    description: |
+      The resource name of the active directory.
+  - !ruby/object:Api::Type::String
+    name: 'createTime'
+    description: |
+      Create time of the active directory. A timestamp in RFC3339 UTC "Zulu" format. Examples: "2023-06-22T09:13:01.617Z".
+    output: true
+  - !ruby/object:Api::Type::Enum
+    name: 'state'
+    description: |
+      The state of the Active Directory policy (not the Active Directory itself).
+    values:
+      - :STATE_UNSPECIFIED
+      - :CREATING
+      - :READY
+      - :UPDATING
+      - :DELETING
+      - :IN_USE
+      - :ERROR
+    output: true
+  - !ruby/object:Api::Type::String
+    name: 'domain'
+    description: |
+      Fully qualified domain name for the Active Directory domain.
+    required: true
+  - !ruby/object:Api::Type::String
+    name: 'site'
+    description: |
+      Specifies an Active Directory site to manage domain controller selection.
+      Use when Active Directory domain controllers in multiple regions are configured. Defaults to `Default-First-Site-Name` if left empty.
+    required: false
+  - !ruby/object:Api::Type::String
+    name: 'dns'
+    description: |
+      Comma separated list of DNS server IP addresses for the Active Directory domain.
+    required: true
+  - !ruby/object:Api::Type::String
+    name: 'netBiosPrefix'
+    description: |
+      NetBIOS name prefix of the server to be created.
+      A five-character random ID is generated automatically, for example, -6f9a, and appended to the prefix. The full UNC share path will have the following format:
+      `\\NetBIOS_PREFIX-ABCD.DOMAIN_NAME\SHARE_NAME`
+    required: true
+  - !ruby/object:Api::Type::String
+    name: 'organizationalUnit'
+    description: |
+      Name of the Organizational Unit where you intend to create the computer account for NetApp Volumes.
+      Defaults to `CN=Computers` if left empty.
+    required: false
+    default_from_api: true
+  - !ruby/object:Api::Type::Boolean
+    name: 'aesEncryption'
+    description: |
+      Enables AES-128 and AES-256 encryption for Kerberos-based communication with Active Directory.
+    required: false
+  - !ruby/object:Api::Type::String
+    name: 'username'
+    description: |
+      Username for the Active Directory account with permissions to create the compute account within the specified organizational unit.
+    required: true
+  - !ruby/object:Api::Type::String
+    name: 'password'
+    description: |
+      Password for specified username.
```

Comment:
Since we added ignore_read to this parameter, TF cannot pick up manual changes done to this parameter and fix them. We should add that to the documentation. Maybe add the following sentence to the description:
"Manual changes done to the password will not be detected. Terraform will not re-apply the password, unless you use a new password in Terraform."
AFAICT, the easiest was for a user would be to change the password in TF to some fake value, do an apply, change it to the correct value and do apply again. That will cause TF to write the correct password back to API.